.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# default group
+* @pcdshub/epics-reviewers
```

Comment:
this is another caproto IOC with no actual code in the repo
maybe this should be xbd (alone, or in addition to?)

Reply:
I think I put python filetypes in addition on the last caproto ioc, but in reality those filters will never trigger.  I'll remove the language specific files here.

Ensuring consistency between all 1000 of these is something I've given up on

Reply:
I agree that full consistency should not be the goal, the goal should be making sure the people who get pinged are close to correct to the point where they could ping the right people or adjust the codeowner files themselves later as needed